translations/en.json

```diff
@@ -2435,5 +2435,8 @@
     "share-opensea-link": "Share OpenSea link",
     "save-image-to-photos": "Save image to Photos",
     "copy-all-details": "Copy all details",
-    "share-details": "Share details"
+    "share-details": "Share details",
+    "sending...": "Sending...",
```

Comment:
Generally, we don't use ellipsis in the keys. Should we rename it to `sending-with-ellipsis` or something?

Reply:
sure, will adjust 👍